src/main/java/org/lightcouch/View.java

```diff
@@ -583,7 +585,14 @@ public View tempView(MapReduce mapReduce) {
     }
 
     private String getKeyAsJson(Object... key) {
-        return (key.length == 1) ? gson.toJson(key[0]) : gson.toJson(key); // single or complex key
+        // single or complex key
+        if (key.length == 1 && (key[0] instanceof Boolean || key[0] instanceof Number)) {
+            return gson.toJson(key);
+        } else if (key.length == 1) {
+            return gson.toJson(key[0]);
+        } else {
+            return gson.toJson(key);
+        }
```

Comment:
@mikerhodes I realized after this addition above that the following start/end keys are valid and can effect the pagination: [1], ["1"], ["abc"].  I added a note in FB that I was in progress of fixing this.  The idea would be to check if the key is an array containing one value, and then manipulating the value before setting the final page parameters for start/end key.